### ISSUE TITLE
[BUG] "Emissive textures don't work when iris is installed"

### ISSUE THREAD

**Issue by deftesthawk** [Traben-0/EmissiveMod#255]
I am using a resource pack that adds emissive armor trims but it wasn't working, after disabling Iris It worked. I read you mod page and it says Iris is supported so I'm assuming this is a bug

Link to resource pack: https://www.curseforge.com/minecraft/texture-packs/xxvis-shiny-armor-trims

- Minecraft version: [1.20.4]
- ModLoader: [fabric-0.15.6]
- ETF Version [5.2.3]
- Iris Version [1.6.17]
- The resource pack works with only ETF installed but not when Iris is installed

**Comment by Traben-0** [comment]
Do you have the 3d skin layers mod?

**Comment by deftesthawk** [comment]
> Do you have the 3d skin layers mod?

No, I tested it in a test profile where the only mods installed were Iris, Sodium, ETF and the resource pack. 

I also tested it with Indium and continuity added to that list of mods which made the item version of the armor emissive but not the player model version due to that being an entity

**Comment by Traben-0** [comment]
is it breaking with shaders on or just iris present?

**Comment by deftesthawk** [comment]
> is it breaking with shaders on or just iris present?

It breaks when iris is installed/enabled, I’ve tested it with shaders turned off but it still stops the emissive textures from working. To get them working I had to disable or uninstall the Iris mod

**Comment by Traben-0** [comment]
will be fixed in 6.1
